src/pytest_plugins/filler/filler.py

```diff
@@ -167,8 +167,16 @@ def pytest_addoption(parser: pytest.Parser):
         "--index",
         action="store_true",
         dest="generate_index",
-        default=False,
-        help="Generate an index file for all produced fixtures.",
+        default=True,
```

Comment:
This option now has no-effect, removed in #1130. Keeping it would maintain backward compatibility, but I don't think we need to worry about that yet?

Reply:
PR merged so this is done now.